libs/features/mas/docs/ccd.html

```diff
@@ -44,7 +44,7 @@
       const params = new URLSearchParams(document.location.search);
       const darkTheme = params?.get('theme')?.toLowerCase() === 'dark';
       const theme = document.createElement('script');
-      theme.setAttribute('src', `../../spectrum-web-components/dist/themes/${darkTheme ? 'dark' : 'light'}.js`);
+      theme.setAttribute('src', `../../spectrum-web-components/dist/themes/${darkTheme ? 'darkest' : 'light'}.js`);
```

Comment:
Nit: if we change from 'dark' to 'darkets' we should rename all occurances of 'dark' to 'darkest', e.g. 'darkTheme' and line 66:
`<a href="?theme=dark">Dark</a>`